assessment.js

```diff
@@ -0,0 +1,112 @@
+document.addEventListener("DOMContentLoaded", () => {
+    console.log("DOMContent is loaded")
+    selectTitle()
+    // displaySelectedFilm(film)                                                                                                                                                                                      select", selectTitle)
+    // let reviewInput = document.querySelector("#text-input")
+    // let form = document.querySelector("form")
+    // form.addEventListener("submit", (event) => {
+    //       let textInput = document.querySelector("#text-input")
+    //textInput.innerText = ""
+    let inputBtn = document.querySelector("#btn-input")
+    inputBtn.addEventListener("submit", (reviewSection))
+    //event.preventDefault()
+    //   if (textInput.innerText = "") {
+    //      form.appendChild(textInput)
+    // })
+
+    // console.log("review box", reviewInput)
+    // reviewInput.addEventListener("click",  {
+
+    // })
+})
+
+const selectTitle = () => {
+    let url = "https://ghibliapi.herokuapp.com/films"
+    fetch(url)
+        .then(response => {
+            console.log(response)
+            return response.json()
+        })
+        .then(film  => {
+            console.log(film)
+            displaySelectedFilm(film)
+        })
+        .catch(error => {
+            console.log("there is an error", error)
+        })
+    //console.log("is it the right url?", url)
+}
+
+const displaySelectedFilm = (film) => {
```

Comment:
`displaySelectedFilms()` seems to be creating `option` elements and appending them to the `select` and also creating and appending the title, release year and descriptions of every movie. This two task are different, therefore they should be done in separate functions.